clients/client-python/gravitino/api/model_version_change.py

```diff
@@ -240,3 +251,58 @@ class name followed by the new URI of the model version.
                 A string summary of this URI update operation.
             """
             return f"UpdateUri {self._new_uri}"
+
+    class UpdateAliases:
+        """A model version change to update the aliases of the model version."""
+
+        def __init__(self, aliases_to_add, aliases_to_delete):
+            self.aliases_to_add = set(aliases_to_add)
```

Comment:
It is better changing to `_aliases_xx`, add `_` prefix to the private variables.